chat-plugins/info.js

```diff
@@ -202,7 +203,11 @@ exports.commands = {
 				}
 			});
 		}
-		if (!results.length) return this.errorReply("No results found.");
+		if (!results.length) {
+			let invalidTarget = net.isIP(target.trim()) === 0 && !target.includes('.');
```

Comment:
To be fair, this will only happen if there are actually no results found from the searched target.  With that being said, in theory, this command still has room to "support IPv6" as soon as PS supports it, @Slayer95